🔨Refactor: 게시글 전체 조회 기능 수정

hyunzoo123123 opened the pull request:
## 의도
- 로그인 한 개인 한 명의 게시글을 모아보는 기능을 게시자를 기준으로 조회 할 수 있게 변경

## 주요 변경점
- 로그인 한 아이디 기준으로 전체 게시글을 조회 하는 기능이었는데,
 멤버아이디 입력 하여 전체 게시글 조회 하는 기능으로 변경 하였습니다.

## ETC
- 오타나 실수 발견하시면 적극적으로 말씀 해주세요.
- 의문 점 있으면 꼭 말씀 주세요.

hyunzoo123123 committed:
🔨Refactor: 프로필 조회 기능 수정

MemberController: 멤버 아이디 기준 정보 조회로 기능 변경
MemberService: 멤버 아이디 기준 정보 조회로 기능 변경
MemberServiceImpl: 멤버 아이디 기준 정보 조회로 기능 변경

Resolves: #12

lazzzykim commented on src/main/kotlin/com/teamsparta/buysell/domain/member/service/MemberServiceImpl.kt:
getMember 함수명 보다 내 정보 조회 처럼 조금 더 직관적인 함수명이었으면 좋겠어요!!!

HwangSeungHyeon replied:
로그인한 사람의 프로필 조회 = getMember -> getMyProfile
멤버 ID를 통한, 해당 인원이 작성한 게시물 조회 = getAllPost -> getAllPostByMemberId
회원탈퇴 - pretendDelete -> signOut

hyunzoo123123 committed:
🔨Refactor: 게시글 조회 기능 수정

MemberController: 멤버가 쓴 글 전체 조회로 기능 변경
MemberService: 멤버가 쓴 글 전체 조회로 기능 변경
MemberServiceImpl: 멤버가 쓴 글 전체 조회로 기능 변경

Resolves: #12

codekmj1 reviewed (approved):
확인했습니다

hyunzoo123123 committed:
🔨Refactor:함수명 수정

중의적이거나 난해한 명칭으로 되어 있는 함수명 수정

Resolves: #12

lazzzykim reviewed (approved):
고생하셔씁니다~

HwangSeungHyeon reviewed (approved):
고생 많으셨습니다!
나중에 수정할 내용 있으면 다시 이야기 드리겠습니다!
